requirements/base.txt

```diff
@@ -2,3 +2,5 @@ openpyxl<3.0.0
 django-admin-rangefilter==0.5.1
 django-import-export==1.2.0
 qrcode==6.1
+tablib==2.0.0
```

Comment:
You should add a comment specifying that this are constraints and why they are there. Also try to update the other packages too to the latest compatible version.

Reply:
Ok, i am going to edit it.